new_project/examples/separate_apps/README.md

```diff
@@ -0,0 +1,13 @@
+# Example for swapping Bitcoin for Ether (separate processes) 
+
+An example project that shows how to swap Bitcoin for Ether with COMIT.
+This example differs from the btc_eth example, as the two apps (for each role) need to be executed separately.
```

Comment:
I think this sentence is obsolete as the mentioned example will be removed with #135.
Shall we rename this example afterwards to btc-eth?

Reply:
I guess add to #135 that this one should be renamed?

Reply:
Added it as a comment